src/test/java/spoon/test/query_function/AllLocalVars.java

```diff
@@ -0,0 +1,263 @@
+package spoon.test.query_function;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.function.Consumer;
+
+/*
+ * The values of the fields must be a sequence starting from 1

```

Comment:
OK. Basically, you check the behavior of Java itself.

What about putting it as a real test class itself "JavaBehaviorTest" and annotate all methods with `@Test`? Together with a good Javadoc/warning, comment it should be maintainable.